# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gnome-vfs-extras/samba-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./smb-method.c.orig	2005-09-14 17:17:50.000000000 +0200
+++ ./smb-method.c	2005-09-14 17:26:27.000000000 +0200
@@ -1200,7 +1200,7 @@
 	} else {
 		LOCK_SMB();
 		init_auth (NULL);
-		if (smb_context->close (smb_context, handle->file) < 0) {
+		if (smb_context->close_fn (smb_context, handle->file) < 0) {
 			res = gnome_vfs_result_from_errno ();
 		}
 		UNLOCK_SMB();
